### ISSUE TITLE
Template app does not work on Windows

### ISSUE THREAD

**Issue by odoe** [Esri/arcgis-js-cli#2]
It looks like the css portion of the loaders do not work on Windows.

## Description

Multiple failures while trying to parse the `.scss` of the template app.

## Expected Behavior

It shouldn't fail.

## Actual Behavior

```sh
[./src/css/main.scss] 1.43 KiB {index} [built] [failed] [1 error]
[./src/index.ts] 323 bytes {index} [built]
    + 2032 hidden modules

ERROR in ./src/css/main.scss
Module build failed: ModuleParseError: Module parse failed: Unexpected character ' ' (1:0)
```

## Possible Fix

Nothing yet.

## Steps to Reproduce

1. `arcgis create myapp`
2. `cd myapp`
3. `npm start`

## Your Environment

* Version used: `4.7.2`
* Browser Name and version: `na`
* Operating System and version (desktop or mobile): Windows 10
* Link to your project if available: `na`

**Comment by odoe** [comment]
Looks like my attempt to scope the assets to only `arcgis-js-api` doesn't work in Windows.
https://github.com/Esri/arcgis-webpack-plugin/commit/641614ecc3f7debd7b867ce136fe96af8d4490b5#diff-168726dbe96b3ce427e7fedce31bb0bcL107

Since this regex doesn't work in Windows, when Webpack hits one of these font or image files, it doesn't have a loader available for that particular file type.

If someone knows how to have this regex work in all environments, I'm open to suggestions. @tomwayson any ideas? _I blame you_ 😛 It's a good suggestion, just not sure why Windows is being weird about it.

I'll revert this in next update, but I have other features currently in the pipeline for the next release.

**Comment by tomwayson** [comment]
What terminal client in Windows? Like I don't think we should support PowerShell.

**Comment by odoe** [comment]
I have no clue, I don't Windows.

Just tested these on my Win10 machine at home and they work.

`/arcgis-js-api[\\\/].*(jpe?g|png|gif|webp)$/`

`/arcgis-js-api[\\\/].*(wsv|ttf|otf|eot|svg|woff(2)?)(\?[a-z0-9]+)?$/`

I'll test on Mac when I get in the office.

**Comment by odoe** [comment]
That was in Powershell. Is it different from command prompt? I'll try that right now.

**Comment by odoe** [comment]
Ok, the above worked in command prompt, I think we're good there.

**Comment by tomwayson** [comment]
I think we should only support bash-like clients (like cigwin, git-bash) and maybe the Linux subsystem on Windows. Not DOS-like clients or PowerShell.

**Comment by odoe** [comment]
If it gets weird, I'll add that to the plugin doc

**Comment by tomwayson** [comment]
Also, I don't think this is part of the current issue, but I think when you scoped the RegExs, I think you changed the meaning:

https://github.com/Esri/arcgis-webpack-plugin/commit/641614ecc3f7debd7b867ce136fe96af8d4490b5#diff-168726dbe96b3ce427e7fedce31bb0bcR47

In the original, you're looking for names that end in ".jpeg", etc in the scoped one, you're looking for ones that end in "jpeg", etc.

**Comment by odoe** [comment]
_ugh_, fixing that too

**Comment by odoe** [comment]
This should now be fixed with `@arcgis/webpack-plugin` `4.7.8`.
https://github.com/Esri/arcgis-webpack-plugin/releases/tag/4.7.8

There might be another release in next couple of days, so I'll wait on updating the CLI until that is done.
